packages/devtools-reps/src/reps/test-helpers.js

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { shallow } = require("enzyme");
+const { REPS } = require("./rep");
+const { getGripPreviewItems } = require("./rep-utils");
+const { Rep } = REPS;
+const nodeConstants = require("../shared/dom-node-constants");
+
+class TestHelpers {
```

Comment:
I think we should follow the same pattern we have in other files : have all of the helpers as plain functions and export them (so here we'd get rid of the class)